smartcontractkit/external-adapters-js
# Feature/33003 input param types 1

## Closes #33003

## Description
Add types to inputParameters for each EA (this covers adapters from 1 to O)

......

## Steps to Test

1. Run unit tests & integration tests
2. Run e2e tests
3. Manually run adapters & verify that they work the same way as before

## Quality Assurance
